apiserver/facades/agent/agent/agent_test.go

```diff
@@ -70,7 +71,11 @@ func (s *agentSuite) SetUpTest(c *gc.C) {
 func (s *agentSuite) TestAgentFailsWithNonAgent(c *gc.C) {
 	auth := s.authorizer
 	auth.Tag = names.NewUserTag("admin")
-	api, err := agent.NewAgentAPIV2(s.State, s.resources, auth)
+	api, err := agent.NewAgentAPIV2(facadetest.Context{
```

Comment:
Why not make this a helper on the suite given that `Auth_` appears to be the only thing that differs across the instantiations?

Reply:
I could do, but this is old test code, that probably needs to be re-written anyway.